js/binance.js

```diff
@@ -213,6 +213,7 @@ module.exports = class binance extends Exchange {
                         'margin/crossMarginData': { 'cost': 0.1, 'noCoin': 0.5 },
                         'margin/isolatedMarginData': { 'cost': 0.1, 'noCoin': 1 },
                         'margin/isolatedMarginTier': 0.1,
+                        'margin/rateLimit/order': 0.1334,
```

Comment:
How do you calculate this rateLimit?
This link says it's weight 20 by ip: https://binance-docs.github.io/apidocs/spot/en/#query-isolated-margin-tier-data-user_data
1 IP weight = cost of 0.1
20 IP weights = cost of 2

So it should be:
```
                        'margin/rateLimit/order': 2,
```

Let's make sure that the rate limits for the other newly added endpoints are correct.

Reply:
I calculate in the old way...